test/upgrade/prober/verify.go

```diff
@@ -103,12 +110,14 @@ func (p *prober) fetchExecution(ctx context.Context) *fetcher.Execution {
 	ex := &fetcher.Execution{
 		Logs: []fetcher.LogEntry{},
 		Report: &receiver.Report{
-			State:  "failure",
-			Events: 0,
+			State:         "failure",
+			EventsSent:    0,
+			TotalRequests: 0,
 			Thrown: receiver.Thrown{
 				Unexpected: []string{"Report wasn't fetched"},
 				Missing:    []string{"Report wasn't fetched"},
 				Duplicated: []string{"Report wasn't fetched"},
+				Unavail:    []string{"Report wasn't fetched"},
```

Comment:
Please don't use abbreviations, as they might not be clear, what they mean.

```suggestion
				Unavailable:    []string{"Report wasn't fetched"},
```

Reply:
Fixed as suggested.